views/type-sampler.html

```diff
@@ -0,0 +1,100 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Headache" %}
+
+{% block localheader %}
+
+{% endblock %}
+
+{% block content %}
+
+<div class="content">
+
+  <div class="grid-row">
+
+    <div class="grid-2-3">
+
+      <h1>Heading 1</h1>
+      <h1>Heading that Runs Over Two lines</h1>
+      <hr />
+      <h2>Heading 2</h2>
+      <h2>Heading that Runs Over Two lines if it is long enough</h2>
+      <hr />
+      <h3>Heading 3</h3>
+      <h3>Heading that Runs Over Two lines if it is long enough</h3>
+      <hr />
+      <h4>Heading 4</h4>
+      <h4>Heading that Runs Over Two lines if it is long enough</h4>
+      
+      <p>There are also things you can do yourself to <a href="http://www.nhs.uk/conditions/Indigestion/Pages/Introduction.aspx">
+      help ease indigestion</a> and <a href="http://www.nhs.uk/Conditions/Constipation/Pages/Introduction.aspx">
+      constipation</a></p>
+
+      <p>If you have vomiting and diarrhoea with your stomach ache, it should
+      stop after a few days. Try to drink a lot of fluids to avoid
+      dehydration.</p>
+
+      <p>Painkillers like paracetamol or ibuprofen can help with your stomach
+      ache but won’t stop any vomiting and diarrhoea.</p>
+
+      <p>Speak to your pharmacist about medicine to help with:</p>
+
+      <hr />
+
+      <ul>
+        <li>feeling bloated (trapped wind)</li>
+        <li>indigestion (burning pain after eating, maybe bringing up food)</li>
+        <li>constipation (can’t poo)</li>
+        <ul>
+            <li>feeling bloated (trapped wind)</li>
+            <li>indigestion (burning pain after eating, maybe bringing up food)</li>
+            <li>constipation (can’t poo)</li>
+        </ul>
+      </ul>
+
+      <hr />
+
+      <ol>
+        <li>feeling bloated (trapped wind)</li>
+        <li>indigestion (burning pain after eating, maybe bringing up food)</li>
+        <li>constipation (can’t poo)</li>
+        <ol>
+            <li>feeling bloated (trapped wind)</li>
+            <li>indigestion (burning pain after eating, maybe bringing up food)</li>
+            <li>constipation (can’t poo)</li>
+        </ol>
+      </ol>
+
+
+<!--       <h1>Heading 1</h1>
+      <p>You can simply wait until the headache goes away or take painkillers
+        like paracetamol or ibuprofen. You can buy these over the counter.</p>
+      <p>Headaches can last between 30 minutes and several hours.</p> -->
+
+    </div>
+<!--     <div class="grid-1-3">
```

Comment:
May as well just drop this content all together instead of leaving commented out